### PR TITLE
Move all tests to testscript 

### PR DESCRIPTION
**Pull request by thiagonache**
All tests are done via testscript now.

### CODE REVIEW

File: ratchet/Makefile [hunk 1]
```diff
@@ -8,15 +8,15 @@ schemata/%.json.zst:
...
 
-.PHONY: clean
-clean:
-	rm -v output/*.cue build/*.json schemata/*.json.zst || true
-
-.PHONY: test
-test: clean 
-test: output/registry.terraform.io_integrations_github_5.9.2.cue
-test: output/registry.terraform.io_hashicorp_aws_5.6.1.cue
+testdata/script/unit/datasources/%.txtar: SOURCE=$(shell echo "$@" | sed "s/datasources/resources/")
```
Comment (jpluscplusm) [inline ratchet/Makefile]: `SOURCE` doesn't appear to be used in this target's recipe.

File: ratchet/Makefile [hunk 2]
```diff
@@ -8,15 +8,15 @@ schemata/%.json.zst:
 build/%.json: schemata/%.json.zst
 	zstdcat "$^" >"$@"
 
-output/%.cue: ADDRESS=$(shell echo "$*" | tr '_' '/' | cut -d/ -f1-3)
-output/%.cue: build/%.json
-	go run cmd/ratchet/main.go "$^" $(ADDRESS) >"$@"
+testdata/script/unit/datasources/%.txtar.tmp: SOURCE=$(shell echo "$@" | sed -e "s/datasources/resources/" -e "s/.tmp//")
```
Comment (jpluscplusm) [inline ratchet/Makefile]: I /feel/ like this target's recipe could easily be rolled into the recipe of the non-`.tmp` version that uses this file, thereby making the flow clearer. I'm not 100% sure, as I don't have any insight into why these recipes are doing what they're doing.

File: ratchet/testdata/bin/sync_testscript_files.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+for SRC_FILE in $(ls testdata/script/unit/resources/*.txtar);
```
Comment (jpluscplusm) [inline ratchet/testdata/bin/sync_testscript_files.sh]:
```suggestion
for SRC_FILE in testdata/script/unit/resources/*.txtar; 
```

**Review by jpluscplusm — APPROVED** [review]
LGTM :-) Resolve all the conversations that I began however suits you best - my suggestions/etc don't /need/ to be incorporated!

File: ratchet/Makefile (hunk 2 above)
Comment (thiagonache) [reply ratchet/Makefile]: yeah. can we pair to improve this makefile?

File: ratchet/testdata/bin/sync_testscript_files.sh (hunk 3 above)
Comment (thiagonache) [reply ratchet/testdata/bin/sync_testscript_files.sh]: done

File: ratchet/Makefile (hunk 1 above)
Comment (thiagonache) [reply ratchet/Makefile]: fixed

**Pull request merged**